teste2e/test_format_upgrades.py

```diff
@@ -0,0 +1,127 @@
+from util import UrsadbTestContext
+from util import ursadb  # noqa
+import json
+import subprocess
+import time
+from pathlib import Path
+import tempfile
+import os
+import zmq
+import json
+from typing import Dict, Any
+
+
+class TemporaryStorage:
+    def __init__(self):
+        self.tmpfiles = []
+        self.tmpdir = tempfile.gettempdir()
+
+    def tmpfile(self, name=None) -> Path:
```

Comment:
```suggestion
    def tmpfile(self, name: Optional[str] = None) -> Path:
```

This requires to `from typing import Optional`!